addusers.py

```diff
@@ -285,7 +256,13 @@ def parse_rows(rows):
     return projects, bad_rows
 
 if __name__ == "__main__":
-    openstack = Openstack(admin_user, admin_pwd, admin_tenant, auth_url, nova_version)
+    auth = v3.Password(auth_url=auth_url,
+                       username=admin_user,
+                       user_domain_id = 'default',
+                       password=admin_pwd)
```

Comment:
Right. you should be able to have a valid auth and you have a "unscoped" (in terms of project scoped) session I believe. Not sure if this fact would affect other clients such as nova. I am thinking, in order to use nova or other clients, you might want to have a scoped session?